invenio_records_lom/fixtures/demo.py

```diff
@@ -382,26 +380,107 @@ def create_fake_record(fake: Faker):
     else:
         fake_embargo = {}
 
-    fake_access = {
+    return {
         "files": fake_access_type,
         "record": fake_access_type,
         "embargo": fake_embargo,
     }
 
-    data = {
+
+def create_fake_data(fake: Faker, resource_type: str, files_enabled: bool = False):
+    """Create a fake json of an invenio-record, "metadata" conforms to LOM-standard."""
+    resource_types = current_app.config["LOM_RESOURCE_TYPES"]
+    resource_type = resource_type or fake.random.choice(resource_types)
+    return {
         # these values get processed by service.config.components
-        "access": fake_access,
+        "access": create_fake_access(fake),
+        "files": {"enabled": files_enabled},
         "metadata": create_fake_metadata(fake),
+        "resource_type": resource_type,
     }
 
-    service = current_records_lom.records_service
-    draft = service.create(data=data, identity=fake_identity)
-    service.publish(id_=draft.id, identity=fake_identity)
-
 
-def create_fake_records(number: int, seed: int = 42) -> list:
+def publish_fake_record(fake: Faker):
```

Comment:
please don't use nested functions in such a context. The outer function publish_fake_record gets to long and therefore difficult to understand.